src/main/java/training/supportbank/transactionhandlers/ExtractNames.java

```diff
@@ -0,0 +1,21 @@
+package training.supportbank.transactionhandlers;
+
+import training.supportbank.models.Transaction;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+//Provides static ExtractNames functions
+public class ExtractNames {
+    public static Set<String> getUniqueNames(List<Transaction> transactions) {
+        Set<String> accountNames = new HashSet<>();
+
+          for(Transaction transaction : transactions) {
```

Comment:
Something weird going on with the indentation here. The logic looks good though. 👍